MiniSim/MenuItems/IOSSubMenuItem.swift

```diff
@@ -11,6 +11,7 @@ enum IOSSubMenuItem: Int, CaseIterable {
     
     case copyName = 100
     case copyUDID = 101
```

Comment:
If you could also add the separator item (same as for the android submenu). This will make sure that user won't click the delete button by accident.